lib/files.cf

```diff
@@ -778,7 +778,15 @@ bundle edit_line set_line_based(v, sep, bp, kp, cp)
       unless => strcmp( "true", $(edit.empty_before_use) );
 @endif
 
-    (cfengine_3_15|cfengine_3_16|cfengine_3_17|cfengine_3_18|cfengine_3_19|cfengine_3_20)::
+@if minimum_version(3.18)
```

Comment:
@craigcomstock note this change to `set_line_based`, Ill need to pick it back to master.

Reply:
Here https://github.com/cfengine/masterfiles/pull/2443